requirements/base.in

```diff
@@ -3,6 +3,6 @@
 
 Django                                  # Web application framework
 # openedx-events 3.1.0 defines producer interface
-openedx-events>=3.1.0                   # Events API
+openedx-events>=4.2.0                   # EventsMetadata default fields
```

Comment:
I think we'd want to update the line above instead -- the line-end comment should still describe the library.

(I'd leave a suggestion diff, but GitHub isn't letting me do that—something bogus about not allowing commenting on deleted lines.)